### DISCUSSION TITLE
Add element duplicated, not delete it

### DISCUSSION THREAD

**Discussion by radikris** [koolamusic/chakra-ui-autocomplete#95] [General]
Hi!

Thanks for this awesome lib.
We would like to have this option to add an item duplicated to the list, there should not be an icon, because now the default behaviour is that the item will be deleted automatically from the selected list.

Thanks for your cooperation!
Happy hacktoberfest!

**Comment by MartonGallyas** [comment]
sounds cool, this would be so helpful for my project! cant wait for it! 😍

**Comment by koolamusic** [comment]
Hi, just reading over this now and it's a bit confusing. 
Could you share an example of what it is you want to achieve. 
How it should work, from a `code` perspective would also help

**Comment by radikris** [comment]
![image](https://user-images.githubusercontent.com/55052220/137644961-d5a118af-1ea4-4d13-8c34-0e599945d79f.png)

I would like to add for example 2 pieces of Order2 to my list. But if I click again, it will be deleted automatically.
We would like this behaviour if you could disable this autodelete feature.

![image](https://user-images.githubusercontent.com/55052220/137644986-04308b06-27bb-42d9-96dc-a428324ff4e9.png)

Hope it is now more understandable!
Let me know if you have any other questions regarding to this.

**Comment by koolamusic** [comment]
Ahaa. 
You're referring to the ability to set select type to `Multiple` instead of defaulting to only `Single` selection yeah?

**Comment by radikris** [comment]
Yes, exactly. This way it would be possible to have duplicated items in the list.

**Comment by koolamusic** [comment]
Feel free to send a PR, and I can review
